frontend/src/app/PrimeErrorBoundary.tsx

```diff
@@ -51,7 +51,9 @@ export default class PrimeErrorBoundary extends React.Component<
         process.env.REACT_APP_OKTA_URL
       }/oauth2/default/v1/authorize?client_id=${
         process.env.REACT_APP_OKTA_CLIENT_ID
-      }&redirect_uri=${getUrl()}&response_type=token id_token&scope=openid simple_report simple_report_test&nonce=thisisnotsafe&state=thisisbogus`;
```

Comment:
Not sure if this is what @bmaden-usds was referencing but doesn't this need to be URL encoded?

Reply:
Just pushed a commit to add URL encoding :+1: